### PR TITLE
update: view submission with new file names

### PR DESCRIPTION
**Pull request by jeanbez**
This updates the view page of a submission to also display the output files considering the new naming format. For instance, `ior_easy_read` was renamed to `ior-eay-read` in the latest version of the benchmark. This update will filter the files considering both names.

### CODE REVIEW

**Review by johnbent — APPROVED** [review]

**Review by JulianKunkel — APPROVED** [review]

**Review by adilger — APPROVED** [review]
Minor typo in the commit message:

    ìor-eay-read

has "accent-i" instead of "`ior-..." and "eay" instead of "easy"...

**Comment by gflofst** [comment]
I don't like this. It should invisibly select both.

**Comment by jeanbez** [comment]
> I don't like this. It should invisibly select both.

We're not renaming the files, only filtering by both naming conventions.

**Comment by gflofst** [comment]
I understand. What I am asking is if there is a reason we need to differentiate between the two? Otherwise, selecting ior-read-easy should also return ior_read_easy. It is what people expect. Computers are supposed to hide ugly things like this from people rather than forcing them to deal with it.

**Comment by jeanbez** [comment]
Makes sense! I've just updated the code to transparently handle both file names.

**Comment by gflofst** [comment]
Thanks. That looks great.

**Review by gflofst — APPROVED** [review]

**Review by johnbent — APPROVED** [review]

**Review by adilger — APPROVED** [review]
Please merge with "squash commits"

**Review by gflofst — APPROVED** [review]

**Comment by jeanbez** [comment]
@adilger Could you please enable option in GitHub so we can do it automatically on PRs?

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-commit-squashing-for-pull-requests

**Comment by adilger** [comment]
There isn't anything to be done, AFAIK?  Once a patch has enough approvals, the "Merge" button will be active, and the arrow to the right will allow selecting between "Merge", "Rebase and Merge" (preferred for single patches, since it avoids making an extra commit in the repo for a PR if it wasn't based on the tip of the tree), and "Squash and Merge" (what we want for patch series like this one).

**Review by gmarkomanolis — APPROVED** [review]

**Comment by jeanbez** [comment]
@adilger Yes, but there is an option to enforce a linear commit history on the main branch, so PR are always squashed. I thought that might be helpful based on your suggestion to organize the repository commit history :)

**Review by JulianKunkel — APPROVED** [review]

**Pull request merged**
